app/bundles/LeadBundle/Controller/AjaxController.php

```diff
@@ -313,62 +311,6 @@ public function clearSocialProfileAction(Request $request)
         return $this->sendJsonResponse($dataArray);
     }
 
-    /**
-     * Updates the timeline events and gets returns updated HTML.
-     *
-     * @return \Symfony\Component\HttpFoundation\JsonResponse
-     */
-    public function updateTimelineAction(Request $request)
```

Comment:
I don't see where is this action used either, but how is removing it related to the goal of this PR?

Reply:
Ah, I see. It's calling `history.html.twig` that does not exist.